packages/components/src/icon/stories/index.story.tsx

```diff
@@ -7,44 +7,68 @@ import type { Meta, StoryFn } from '@storybook/react';
  * WordPress dependencies
  */
 import { SVG, Path } from '@wordpress/primitives';
-import { wordpress } from '@wordpress/icons';
+import * as icons from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
-import Icon from '..';
+import Icon, { type Props, type IconType } from '..';
 import { VStack } from '../../v-stack';
 
-const meta: Meta< typeof Icon > = {
+type IconProps = Props & {
+	additionalProps: object;
+};
+
+const meta: Meta< IconProps > = {
 	title: 'Components/Icon',
 	component: Icon,
 	parameters: {
 		controls: { expanded: true },
 		docs: { canvas: { sourceState: 'shown' } },
 	},
+	args: {
+		additionalProps: {},
+		size: 24,
+	},
+	argTypes: {
+		size: {
+			control: {
+				type: 'range',
+				min: 1,
+				max: 200,
+			},
+		},
```

Comment:
Similar to the discussions we've been having about range inputs in general (#60633), but range inputs imply that a min/max exists, which is untrue in this case. A standard number input is more in line with all the other props controls in our Storybook.

Reply:
I understand that Storybook is primarily a tool for users to discover and try components easily. The standard number component is less handy. It defaults to zero without a default set and lets users input negative numbers, which is inconvenient and doesn't reflect the expected use of the component.

Reply:
And yet that reflects the actual component logic the best, because we don't restrict values, neither at runtime, in docs, nor in TypeScript types. I'm not sure if there's a single number type prop in our library where we do restrict values in any of these ways. Probably because it's usually obvious and goes without saying. It wouldn't be worth the effort to assess and uphold these guardrails for every single number prop.

It's not about what's acceptable for a single prop in a single component, but what consistent patterns we can adopt at scale.

Reply:
I agree with Lena here — I understand that the way Storybook controls may feel inconvenient ("I want to set a  custom size and it defaults to 0" or "It lets me set a negative size"), but:

- we greatly value consistency across all Storybook examples;
- a consumer of the component would be able to pass "0" or even a negative number to size in the code anyway